pkgs/by-name/li/libiff/package.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "libiff";
+  version = "0.1-unstable-2024-03-02";
```

Comment:
There is no tagged version so it should be `0` instead of `0.1`.

Reply:
Okay. I was going by the the contents of [the version file](https://github.com/svanderburg/libiff/blob/master/version), but you're right, it's probably not a formal version.